File: fix-services.js
const fs = require('fs');

let content = fs.readFileSync('app/data/services.ts', 'utf8');

// Fix common syntax errors
content = content.replace(/;\s*}/g, '}');
content = content.replace(/,\s*}/g, '}');
content = content.replace(/;\s*]/g, ']');
content = content.replace(/,\s*]/g, ']');
content = content.replace(/;\s*\)/g, ')');
content = content.replace(/,\s*\)/g, ')');

// Fix missing commas before properties
content = content.replace(/(\w+)\s*\n\s*(\w+):/g, '$1,\n    $2:');

// Fix missing commas in arrays
content = content.replace(/(\w+)\s*\n\s*(\w+)/g, '$1,\n    $2');

fs.writeFileSync('app/data/services.ts', content);
console.log('Fixed services.ts syntax errors');

File: fix_services.py
#!/usr/bin/env python3

import re

# Read the file
with open('/workspace/app/data/services.ts', 'r') as f:
    content = f.read()

# Fix the malformed object syntax
# Pattern: users: 'X',}freeTrial: 'Y',},
content = re.sub(r"users: ([^,]+),}freeTrial: ([^,]+),},", r"users: \1,\n    freeTrial: \2\n  },", content)

# Pattern: users: 'X',}freeTrial: 'Y',}
content = re.sub(r"users: ([^,]+),}freeTrial: ([^,]+),}", r"users: \1,\n    freeTrial: \2\n  }", content)

# Write the fixed content back
with open('/workspace/app/data/services.ts', 'w') as f:
    f.write(content)

print("Fixed services.ts file with Python")